modules/oaeservice/manifests/hilary.pp

```diff
@@ -40,6 +40,7 @@
   $web_domain = hiera('web_domain')
   $app_admin_tenant = hiera('app_admin_tenant', 'admin')
   $admin_domain = "${app_admin_tenant}.${web_domain}"
+  $guest_domain = "guest.${web_domain}"
```

Comment:
Is not the guest tenant name configurable?


On 20 September 2015 at 02:21, Nicolaas Matthijs <notifications@github.com>
wrote:

> In modules/oaeservice/manifests/hilary.pp
> <https://github.com/oaeproject/puppet-hilary/pull/169#discussion_r39925794>
> :
>
> > @@ -40,6 +40,7 @@
> >    $web_domain = hiera('web_domain')
> >    $app_admin_tenant = hiera('app_admin_tenant', 'admin')
> >    $admin_domain = "${app_admin_tenant}.${web_domain}"
> > +  $guest_domain = "guest.${web_domain}"
>
> Yeah, it probably makes sense to have them be consistent
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/oaeproject/puppet-hilary/pull/169/files#r39925794>.
>